### DISCUSSION TITLE
Frontend Developer

### DISCUSSION THREAD

**Discussion by shmodarresi** [WeAreMahsaAmini/Core#344] [Introductions]
Hi,

I'm a Frontend developer and working with React js. I can help in web projects.
We will do it ✌️

**Comment by Pink3lephant** [comment]
Hi @shmodarresi 

Nice to hear from you. We are working on a project to distribute knowledge on how to create VPN servers that actually work (think of it as meta-Outline, where you can setup several protocols). 

At the moment we are working on our landing page (Nextjs / Strapi], it's in progress but we need more resources to finish it faster. 

Can you please have a look and let me know if you're interested in helping out? 

🐜  Current website: [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail.
